dotnet/src/Microsoft.AutoGen/Agents/Agents/AIAgent/SKAiAgent.cs

```diff
@@ -7,13 +7,13 @@
 using Microsoft.SemanticKernel.Memory;
 
 namespace Microsoft.AutoGen.Agents;
-public abstract class AiAgent<T> : AgentBase where T : class, new()
+public abstract class SKAiAgent<T> : AgentBase where T : class, new()
```

Comment:
Thanks for renaming this 👍 
When reading through the codebase, it confused me a lot and It made me feel that AutoGen had a force coupling to SK which caused a few concerns and confusion.

Reply:
yes will move it to its own extension in another pr. today it needs to stay there for the dev-team example. HAve opened an issue to track this.